src/language/translations/en-US.json

```diff
@@ -575,7 +575,18 @@
       "header": "Organisation members",
       "actions": "User role",
       "invite": "Invite member",
-      "send": "send"
+      "send": "Send",
+      "options": {
+        "removeUserFromTeam": "Remove user from team"
+      },
+      "dialogs": {
+        "removeUserFromTeam": {
```

Comment:
Redundant name. `rbac > team > dialogs > removeUserFromTeam` change to `removeUser` or just `remove` you are already in the team context.